src/Models/SslCertificateCheck.php

```diff
@@ -50,6 +51,10 @@ public function saveError(Exception $exception): void
         $this->issuer = '';
         $this->check_failure_reason = $exception->getMessage();
 
+        if (Cache::get('ssl_error_occurrence_time') === null) {
+            Cache::put('ssl_error_occurrence_time', now(), 60 * 24);
+        }
```

Comment:
This code should be inside the `SslCertificateCheckService`.

Check out how it works on `UptimeCheckService`: 

https://github.com/taecontrol/moonguard/blob/a840ca45b0fe4517d03de0ebfd2e55b31af2091e/src/Services/UptimeCheckService.php#L138-L153